cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,21 @@
+const { generateUser } = require('../support/generateUser');
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const {
+      userName,
+      email,
+      password
+    } = generateUser();
+
+    cy.visit('/#/register');
+
+    cy.findByPlaceholder('Username').type(userName);
+    cy.findByPlaceholder('Email').type(email);
+    cy.findByPlaceholder('Password').type(password);
+
+    cy.get('.btn, Sign in').click();
 
+    cy.get('.nav-link').should('contain', userName);
```

Comment:
cypress found 5 elements with that class. Try to fix it. Cypress must find exact element which should contain username

Reply:
Fixed